journal/modernized/02-chapter-2.adoc

```diff
@@ -608,8 +610,8 @@ believe to pass through near three hours of painful silence,
 the spring lay so low, and nothing to draw with; the rubbish so great,
 and few faithful laborers to lay their shoulders to the burden!
 Oh! you faithless generation, what will you do in the end?
-will your being the children of faithful Abraham
-work out your salvation f It is sorrowful,
+Will your being the children of faithful Abraham
+work out your salvation if it is sorrowful,
```

Comment:
From checking the original, this one should be changed to:

```
Will your being the children of faithful Abraham work out your salvation?
It is sorrowful, painfully sorrowful, to behold these things,
```
[Here's the original](https://books.google.com/books?id=-UKEm-qXgA4C&pg=PA56&lpg=PA56&dq=%22Will+your+being+the+children+of+faithful+Abraham%22+conran&source=bl&ots=2pWF0rXMEz&sig=ACfU3U1-gyzTMoEPudtZ0dIbcavYIkEp8A&hl=en&sa=X&ved=2ahUKEwi1ldSg4ZDgAhXCh1QKHT9DDKcQ6AEwAHoECAAQAQ#v=onepage&q=%22Will%20your%20being%20the%20children%20of%20faithful%20Abraham%22%20conran&f=false)